projects/kit/components/block-status/block-status.template.html

```diff
@@ -0,0 +1,15 @@
+<div class="t-image">
+    <ng-content select=".image"></ng-content>
+</div>
+
+<div class="t-title">
+    <ng-content select=".title"></ng-content>
```

Comment:
what do you think about

`select="[tuiBlockStatusTitle]"`

`select="[tuiBlockStatusText]"`


`select="[tuiBlockStatusActions]"`

`select="[tuiBlockStatusImage]"`